cmd/installer/root_test.go

```diff
@@ -416,7 +416,7 @@ func TestUpdatePublicIndex(t *testing.T) {
 
 		indexPath := server.URL() + "this-file-does-not-exist"
 
-		err := installer.UpdatePublicIndex(indexPath, Overwrite, Sparse, DownloadPdsc, Concurrency, Timeout)
+		err := installer.UpdatePublicIndex(indexPath, Overwrite, Sparse, DownloadPdsc, false, Concurrency, Timeout)
```

Comment:
Where can we find the test case for 'true' case?

Reply:
see "test add remaining PDSC from index.pidx"